src/core/symbology-ng/qgsrulebasedrendererv2.cpp

```diff
@@ -428,7 +453,7 @@ bool QgsRuleBasedRendererV2::Rule::startRender( QgsRenderContext& context, const
   // Finally they are joined with their parent (this) with AND
   QString sf;
   // If there are subfilters present (and it's not a single empty one), group them and join them with OR
-  if ( subfilters.length() > 1 || subfilters.value( 0 ).trimmed().length() > 0 )
+  if ( subfilters.length() > 1 || !subfilters.value( 0 ).isEmpty() )
```

Comment:
Are you sure we can omit the `trimmed()` here?
[`isEmpty()` returns true if the string has no characters; otherwise returns false.](http://doc.qt.io/qt-4.8/qstring.html#isEmpty)